src/Components/Browser.JS/src/Platform/Logging/ILogger.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I got this from the signalr repo so that we have a consistent experience

Reply:
It's worth considering if we have enough re-used code in JS/TS that we should make modules out of it if we haven't already.

Reply:
Definitely not. We don't want to end like leftpad